dll/win32/iernonce/iernonce.cpp

```diff
@@ -15,7 +15,7 @@
 #include <debug.h>
 
 #include "registry.h"
-
+#include "dialog.h"
```

Comment:
In the `DLL_PROCESS_ATTACH` case below, please add:
```c
DisableThreadLibraryCalls(hinstDLL);
```

Reply:
> I'm also bringing to your attention the existence of a second iernonce exported function called `InitCallback`.

Oh, I indeed saw this before, but I have completely no idea about what's that for at that time...
So I put it aside and almost forgot it 🤔
I guess it's for `CoInitialize`? I will give it some tests.

Reply:
It stops your dll from receiving `DLL_THREAD_ATTACH` calls in `DllMain`.
This is an optimization, since this is hardly ever required.
Usually, only when you want to use per-thread data you need this callback.